### PR TITLE
clean up the environment

### PR DESCRIPTION
**Pull request by diguage**

### CODE REVIEW

File: content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md [hunk 1]
```diff
@@ -193,6 +193,22 @@ When a Pod consumes a PersistentVolume, the GIDs associated with the
 PersistentVolume are not present on the Pod resource itself.
 {{< /note >}}
 
+## Clean up
```
Comment (sftim) [inline content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: @diguage You can use a `{{% capture cleanup %}}` section for this, I think.

Comment (diguage) [reply content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: Let me try it.

Comment (diguage) [reply content/en/docs/tasks/configure-pod-container/configure-persistent-volume-storage.md]: @sftim 
I study the structure of the document. The `{{% capture cleanup %}}` is used in the `Tutorials` part. The `Tasks` part uses the `{{% capture steps %}}`, and the `Clean up` is the last step of a task. So, I follow the style.

**Comment by diguage** [comment]
@tengqm 
Sorry. I add some changes and the robot remove the `lgtm` label.

**Pull request merged**